### PR TITLE
Add multiple objects method to Client

### PR DESCRIPTION
**Pull request by pengend**

### CODE REVIEW

**Comment by rayrutjes** [comment]
Hey @pengend thanks for your contributions.
It looks like this endpoint is definitely missing as of today.
We will review your PR next week.

Sorry about the delay!

File: src/AlgoliaSearch/Client.php [hunk 1]
```diff
@@ -264,6 +264,50 @@ public function multipleQueries($queries, $indexNameKey = 'indexName', $strategy
         );
     }
 
+    /**
+     * This method allows to get multiple objects from indexes with one API call.
```
Comment (maxiloc) [inline src/AlgoliaSearch/Client.php]: Comment is not good

**Comment by julienbourdeau** [comment]
See #463

**Comment by julienbourdeau** [comment]
Thank you again @pengend

**Pull request closed without merging**